normandy/control/static/control/js/routes.js

```diff
@@ -22,7 +22,7 @@ export default (
       component={RecipeForm}
       pageTitle="Edit Recipe"
       ctaButtons={[
-        {text: 'Preview', icon: 'eye', link: 'preview/'},
+        {text: 'Preview', icon: 'eye', link: 'preview/?testing=true'},
```

Comment:
Oh, because your other commits removed the modification to runRecipe that made the testing parameter work. Could you include that back in?